vm-bootstrap.sh

```diff
@@ -21,7 +21,19 @@ fi
 
 cd /opt/Espressif/crosstool-NG
 
-function gitsha() { git show --format=%H | head -1; }
+# Git complains if these values are not set
```

Comment:
i don't think this git config is necessary -- it's needed for commits, but we're only doing clones and pulls